api/src/main/resources/liquibase.xml

```diff
@@ -4,6 +4,345 @@
                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+    <changeSet author="slubwama" id="patientflags-1554735240623-1">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <tableExists tableName="patientflags_displaypoint"/>
+            </not>
+        </preConditions>
+        <comment>Create Table patientflags_displaypoint</comment>
+        <createTable tableName="patientflags_displaypoint">
+            <column autoIncrement="true" name="displaypoint_id" type="INT">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="name" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="description" type="VARCHAR(1000)"/>
+            <column defaultValueNumeric="0" name="creator" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="date_created" type="datetime">
+                <constraints nullable="false"/>
+            </column>
+            <column name="changed_by" type="INT"/>
+            <column name="date_changed" type="datetime"/>
+            <column defaultValueNumeric="0" name="retired" type="TINYINT(3)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="retired_by" type="INT"/>
```

Comment:
Is there a reason why for some types you are using capital letters and for others small?